Tests/conf.json

```diff
@@ -471,6 +471,10 @@
         {
             "integrations": "TruSTAR",
             "playbookID": "TruSTAR Test"
+        },
+        {
+            "integrations": "MineMeld",
+            "playbookID": "minemeld_test"
```

Comment:
It's the wrong place.. notice that you under "skipped" key on the JSON. plz add it under "tests" key (first part of the JSON)